packages/i18n-calypso/src/number-format.js

```diff
@@ -6,8 +6,7 @@
  * @see https://github.com/kvz/phpjs/blob/ffe1356af23a6f2512c84c954dd4e828e92579fa/functions/strings/number_format.js
  */
 function toFixedFix( n, prec ) {
-	const k = Math.pow( 10, prec );
-	return '' + ( Math.round( n * k ) / k ).toFixed( prec );
+	return '' + Number( Math.round( +n + `e+${ prec }` ) + `e-${ prec }` ).toFixed( prec );
 }
```

Comment:
I've taken a run here at solving the core issue with rounding numbers such as `1.005`. Using scientific notation makes JavaScript play nicely with proper rounding. I'm trying to find an authoritative resource that explains _why_ this is, but haven't come up with anything aside from some [SO answers](https://stackoverflow.com/a/19722641).